foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_8_0/AiopsReferencePythonMigration.java

```diff
@@ -0,0 +1,88 @@
+package com.boozallen.aissemble.upgrade.migration.v1_8_0;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.upgrade.migration.AbstractAissembleMigration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.technologybrewery.baton.BatonException;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.util.*;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import static com.boozallen.aissemble.upgrade.util.FileUtils.replaceLiteralInFile;
+
+public class AiopsReferencePythonMigration extends AbstractAissembleMigration {
+
+    public static final Logger logger = LoggerFactory.getLogger(AiopsReferencePythonMigration.class);
+    public static final Map<String, String> AIOPS_REFERENCE_PYTHON_PACKAGE_MAP = Map.of(
+            "aiops.authority", "aissemble.authority",
+            "aiopskey" , "aissemblekey",
+            "/deployments/aiops-secure.jks" , "/deployments/aissemble-secure.jks",
+            "policy-decision-point.aiops-security.properties.vm", "policy-decision-point.aissemble-security.properties.vm",
+            "aiops_core_filestore.file_store_factory","aissemble_core_filestore.file_store_factory",
+            "aiops_core_metadata.metadata_model", "aissemble_core_metadata.metadata_model",
+            "aiops_core_metadata.metadata_api", "aissemble_core_metadata.metadata_api",
+            "aiops_security.pdp_client", "aissemble_security.pdp_client",
+            "AiopsSecurityException", "AissembleSecurityException",
+            "aiopsauth.auth_config", "aissembleauth.auth_config"
+    );
+
+    /**
+     * Function to check whether the migration is necessary.
+     * @param file file to check
+     * @return shouldExecute - whether the migration is necessary.
+     */
+    @Override
+    protected boolean shouldExecuteOnFile(File file) {
+        System.out.println("Inside should execute on file......*******");
```

Comment:
S: should not be using `System.out.println` - use logger instead (or just remove).

If this file will be the target of the follow on ticket, recommend a preemptive comment to let the reviewer know.